CIPs/cip-0008.md

```diff
@@ -1,8 +1,14 @@
-# CIP 0008: Expand Metadata to general off-chain storage
-
-- Date: 2020-05-05
-- Author: @nambrot
-- Status: WIP
+---
+cip: 0008 
+title: Expand Metadata to General Off-Chain Storage
+author: Nam Chu Hoai <@nambrot>
+discussions-to: 
+status: Draft
```

Comment:
What would be the appropriate status for implementation Work in Progress, but "accepted" as a standard?